bigtable/bigtable.go

```diff
@@ -453,9 +461,53 @@ func prefixSuccessor(prefix string) string {
 	return string(ans)
 }
 
+// ReadIterationStats captures information about the iteration of rows or cells over the course of
+// a read, e.g. how many results were scanned in a read operation versus the results returned.
+type ReadIterationStats struct {
+	CellsReturnedCount int64
+	CellsSeenCount     int64
+	RowsReturnedCount  int64
+	RowsSeenCount      int64
+}
+
+// RequestLatencyStats provides a measurement of the latency of the request as it interacts with
+// different systems over its lifetime, e.g. how long the request took to execute within a frontend
+// server.
+type RequestLatencyStats struct {
+	FrontendServerLatency time.Duration
+}
+
+// FullReadStats captures all known information about a read.
+type FullReadStats struct {
+	ReadIterationStats  ReadIterationStats
+	RequestLatencyStats RequestLatencyStats
+}
+
+func makeFullReadStats(reqStats *btpb.RequestStats) FullReadStats {
+	statsView := reqStats.GetFullReadStatsView()
+	readStats := statsView.ReadIterationStats
+	latencyStats := statsView.RequestLatencyStats
+	return FullReadStats{
+		ReadIterationStats{readStats.CellsReturnedCount, readStats.CellsSeenCount, readStats.RowsReturnedCount, readStats.RowsSeenCount},
```

Comment:
NIT: Preferred style for Go is to instantiate an object by providing explicit field assignments.

E.g.

```
return FullReadStats {
    CellsReturnedCount: readStats.CellsReturnedCount,
    // etc etc
}
```

Reply:
Done.